### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by shm04**
### In this pull request i add the following changes:

- Implement the design from the sneak peek wireframes.
- Use methods that you have created in the Blog App - processing data in models

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,40 @@
...
+  <strong>Number of posts:</strong> <%= @user.post_counter %>
+</div>
+
+<div>
+  <strong>Bio:</strong> <%= @user.bio %>
+</div>
+
+<div>
+  <ul>
+    <% @user.posts.each do |post| %>
```
Comment (t-yanick) [inline app/views/users/show.html.erb:19]: - [ ] Great job working on this milestone. Here, I think you should display just the first 3 posts. The button `All Posts` will now redirect to the page that displays all posts. You should be using the `recent_posts` method that you created in the `user.rb model`
`<% @user.recent_post.each do |post| %>`
- **All Post** link should open user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+
+        <!-- Display the number of likes -->
+        <p><strong>Likes:</strong> <%= post.likes.count %></p>
+      </li>
+    <% end %>
+  </ul>
+</div>
+
+<div>
+  <%= link_to "All Posts", user_posts_path(@user), class: "btn btn-primary" %>
```
Comment (t-yanick) [inline app/views/users/show.html.erb:36]: - All Post link should open user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,7 +1,32 @@
...
+    <h2><%= link_to post.title, user_post_path(@user, post) %></h2>
+    <p><%= post.text %></p>
+
+    <p><strong>Comments:</strong> <%= post.comments.count %></p>
+
+    <p><strong>Likes:</strong> <%= post.likes.count %></p>
+
+    <!-- Display comments for this post -->
+    <ul>
+      <% post.comments.each do |comment| %>
```
Comment (t-yanick) [inline app/views/posts/index.html.erb:24]: - [ ] You should use the `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page as per the wireframe. At the moment, you are displaying all comments on the posts index page. The `recent_comments` method was created in the `post` model.

`<% post.recent_comments.each do |comment| %>`

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @shm04 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: No linter errors

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @shm04, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Applied the changes requested from the previous review (✅)) 👏
- Added the Views associated with the Models and Controllers 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**